Cargo.toml

```diff
@@ -64,7 +64,7 @@ default-net = "0.14"
 wol-rs = "1.0"
 flutter_rust_bridge = { version = "1.75", features = ["uuid"], optional = true}
 errno = "0.3"
-rdev = { git = "https://github.com/fufesou/rdev" }
+rdev = { git = "https://github.com/mcfans/rdev" }
```

Comment:
Please merge to https://github.com/fufesou/rdev
@fufesou  please review.

Reply:
I will open a PR for that.

Reply:
Finish this first

Reply:
@fufesou Could you take a look at [this](https://github.com/fufesou/rdev/pull/9)?